include/legacy-util-api.h

```diff
@@ -88,7 +88,7 @@ char *mkperm(char *pattern, const char *oldext, const char *newext);
 #define ASMFILE		".s"		/* asm source */
 
 /* Measures user+system CPU milliseconds that elapse between calls. */
-unsigned long getcpu(void);
+unsigned long flang_getcpu(void);
```

Comment:
I didn't like the old name.  I don't like the new name.  Given that we have the opportunity to pick a good name, how about something that is more descriptive?  And drop the flang_ prefix :)

Reply:
It's not my native language so I'm not good at proposing names either. What about getMillis()? or getCPUMillis()? or getElapsed()?